concepts/conditionals-if/introduction.md

```diff
@@ -1,57 +1,17 @@
 # Introduction
 
-TODO: the content below is copied from the exercise introduction and probably needs rewriting to a proper concept introduction
-
-## conditionals-if
-
-## conditionals-switch
-
-## booleans
-
-Go supports the three logical operators `&&` (AND), `||` (OR), and `!` (NOT).
-
 Conditionals in Go are similar to conditionals in other languages. The underlying type of any conditional operation is the `bool` type, which can have the value of `true` or `false`. Conditionals are often used as flow control mechanisms to check for various conditions. For checking a particular case an `if` statement can be used, which executes its code if the underlying condition is `true` like this:
 
 ```go
 var value string
 
 if value == "val" {
-    // conditional code
+    return "was val"
 }
-```
-
-In scenarios involving more than one case many `if` statements can be chained together using the `else if` and `else` statements.
```

Comment:
Seeing an example for `else if` and `else` is important to understand this concept so I would recommend to not remove this part.

Maybe instead of giving 2 if-only examples you can have one if-only and one "full" version.